lib/model/katex.dart

```diff
@@ -209,11 +209,122 @@ class _KatexParser {
+          if (vlistStyles != null
+            && vlistStyles.filter(heightEm: false) != const KatexSpanStyles()) {
+            throw _KatexHtmlParseError();
+          }
+        } else {
+          throw _KatexHtmlParseError();
+        }
+      }
+
+      if (vlistT.nodes.first
+          case dom.Element(localName: 'span', className: 'vlist-r') &&
+              final vlistR) {
+        if (vlistR.attributes.containsKey('style')) throw _KatexHtmlParseError();
+
+        if (vlistR.nodes.first
+            case dom.Element(localName: 'span', className: 'vlist') &&
+                final vlist) {
+          // Same as above for the second .vlist-r span, .vlist span in first
+          // .vlist-r span will have "height" inline style which we ignore,
+          // because it doesn't seem to have any effect in rendering on
+          // the web.
+          // But also make sure there aren't any other inline styles present.
+          final vlistStyles = _parseSpanInlineStyles(vlist);
+          if (vlistStyles != null
+            && vlistStyles.filter(heightEm: false) != const KatexSpanStyles()) {
+            throw _KatexHtmlParseError();
+          }
+
+          final rows = <KatexVlistRowNode>[];
+
+          for (final innerSpan in vlist.nodes) {
+            if (innerSpan case dom.Element(
+              localName: 'span',
+              nodes: [
+                dom.Element(localName: 'span', className: 'pstrut') &&
+                    final pstrutSpan,
+                ...final otherSpans,
+              ],
+            )) {
+              if (innerSpan.className != '') {
+                throw _KatexHtmlParseError('unexpected CSS class for '
+                  'vlist inner span: ${innerSpan.className}');
+              }
+
+              var styles = _parseSpanInlineStyles(innerSpan);
+              if (styles == null) throw _KatexHtmlParseError();
+              if (styles.verticalAlignEm != null) throw _KatexHtmlParseError();
+              final topEm = styles.topEm ?? 0;
+
+              styles = styles.filter(topEm: false);
+
+              final pstrutStyles = _parseSpanInlineStyles(pstrutSpan);
+              if (pstrutStyles == null) throw _KatexHtmlParseError();
+              if (pstrutStyles.filter(heightEm: false)
+                  != const KatexSpanStyles()) {
+                throw _KatexHtmlParseError();
+              }
+              final pstrutHeight = pstrutStyles.heightEm ?? 0;
+
+              rows.add(KatexVlistRowNode(
+                verticalOffsetEm: topEm + pstrutHeight,
+                debugHtmlNode: kDebugMode ? innerSpan : null,
+                node: KatexSpanNode(
+                  styles: styles,
+                  text: null,
+                  nodes: _parseChildSpans(otherSpans))));
+            } else {
+              throw _KatexHtmlParseError();
+            }
+          }
+
+          // TODO(1716) Handle styling for .vlist-t2 spans
```

Comment:
nit: follow pattern of our other TODO comments by making more explicit this is an issue reference:
```suggestion
          // TODO(#1716) Handle styling for .vlist-t2 spans
```